templates/docs/examples/utilities/table-layout/table-layout-default.html

```diff
@@ -0,0 +1,21 @@
+{% extends "_layouts/examples.html" %}
```

Comment:
We have enough examples of default tables. I don't think there is a need to this one.

Reply:
Got it; removed that example from the PR